README.md

```diff
@@ -20,6 +20,18 @@ swift-express init YourProject
 * `--template git-url` allows specifying the project template git URL. Defaults to `https://github.com/crossroadlabs/ExpressTemplate.git`.
 * `--path path/to/dir` specifies where to create the application. Defaults to the current directory.
 
+### Install project dependencies
+
+This one for downloading project dependencies (frameworks and libs)
+
+```sh
+swift-express build-deps
```

Comment:
How does it sound to you to make a slight API change, like: `swift-express dependencies` with optional key `--fetch`. 
This way the default behaviour will fetch and build the dependencies. With `--fetch` it will just fetch em without building.
Also, it will make it consistent with SPM.